pkgs/development/python-modules/parsimonious/default.nix

```
@@ -17,6 +17,9 @@ buildPythonPackage rec {
   checkInputs = [ nose ];
   propagatedBuildInputs = [ six ];
 
+  # performance tests tend to fail sometimes
+  checkPhase = "NOSE_EXCLUDE=test_benchmarks nosetests -s";
```

No good - you override the phase completely and not running hooks. How about:

```suggestion
  NOSE_EXCLUDE = "test_benchmarks";
```